client/landing/gutenboarding/components/domain-picker/index.tsx

```diff
@@ -59,12 +59,47 @@ export interface Props {
 	currentDomain?: DomainSuggestion;
 }
 
-const DomainPicker: FunctionComponent< Props > = ( {
-	onDomainSelect,
-	onDomainPurchase,
-	onClose,
-	currentDomain,
-} ) => {
+const FreeDomainIcon = () => (
+	<Icon
+		icon={ () => (
+			<svg
+				width="15"
+				height="15"
+				viewBox="0 0 15 15"
+				fill="none"
+				xmlns="http://www.w3.org/2000/svg"
+			>
+				<path
+					d="M13.2878 8.60833L8.61291 13.2844C8.4918 13.4057 8.34798 13.5019 8.18968 13.5676C8.03137 13.6332 7.86169 13.667 7.69032 13.667C7.51895 13.667 7.34926 13.6332 7.19096 13.5676C7.03265 13.5019 6.88884 13.4057 6.76773 13.2844L1.16699 7.68876V1.16699H7.68706L13.2878 6.76919C13.5307 7.01358 13.667 7.34417 13.667 7.68876C13.667 8.03335 13.5307 8.36395 13.2878 8.60833V8.60833Z"
+					stroke="#008A20"
+					strokeWidth="1.5"
+					strokeLinecap="round"
+					strokeLinejoin="round"
+				/>
+				<circle cx="4.50033" cy="4.50033" r="0.833333" fill="#008A20" />
+			</svg>
+		) }
+	/>
+);
+
+const SearchIcon = () => (
+	<Icon
+		icon={ () => (
+			<svg
+				width="24"
+				height="24"
+				viewBox="0 0 24 24"
+				fill="none"
+				xmlns="http://www.w3.org/2000/svg"
+			>
+				<path d="M6 18L10 14.5" stroke="black" strokeWidth="1.5" />
+				<circle cx="13.5" cy="11.5" r="4.75" stroke="black" strokeWidth="1.5" />
+			</svg>
+		) }
+	/>
+);
```

Comment:
Could maybe use the [search icon](https://github.com/WordPress/gutenberg/blob/a97f31c4a80f4d945d10cfd6a7a5301baad9dabf/packages/icons/src/library/search.js) from the `@wordpress/icons` npm

Reply:
https://github.com/Automattic/wp-calypso/pull/40742/files#r404025438
Tackled in #40803